conjure-java-core/src/main/java/com/palantir/conjure/java/types/AliasGenerator.java

```diff
@@ -120,7 +119,7 @@ public static JavaFile generateAliasType(
             spec.addMethod(MethodSpec.methodBuilder("valueOf")
                     .addModifiers(Modifier.PUBLIC, Modifier.STATIC)
                     .addParameter(ParameterSpec.builder(String.class, "value")
-                            .addAnnotations(safetyAnnotations)
+                            .addAnnotations(computedSafetyAnnotations)
```

Comment:
This is the change I mentioned above, noticed while making changes.

Reply:
I was thinking of making this change but I wasn't sure if this breaks the case where we're aliasing a type that declares its own safety (e.g. reference) - in which case we should be using safetyAnnotations, which will be empty, instead of computedSafetyAnnotations, which would resolve to the safety of the reference type?

Reply:
Oh wait is this desired because value takes a string?

Reply:
Exactly! The input via string-parsing is expected to have the same safety as the thing we're creating, but String.class isn't aware of log-safety.